apps/web/src/pages/settings/tabs/InAppCenterCard.tsx

```diff
@@ -15,7 +15,7 @@ export const InAppCenterCard = () => {
     n[i] = {}; var m = ['init', 'on']; n[i]._c = [];m.forEach(me => n[i][me] = function() {n[i]._c.push([me, arguments])});
     var elt = o.createElement(f); elt.type = "text/javascript"; elt.async = true; elt.src = t;
     var before = o.getElementsByTagName(f)[0]; before.parentNode.insertBefore(elt, before);
-  })(window, document, '${WIDGET_EMEBED_PATH}', 'novu', 'script');
+  })(window, document, '${WIDGET_EMBED_PATH}', 'novu', 'script');
```

Comment:
Rollback this one please. This can affect current users settings and break their systems.